Added postgreSQL installation steps.

imskr opened the pull request:
Solved issue #591

praveenscience commented:
@imskr Did you completely take information from https://www.guru99.com/download-install-postgresql.html and added here?

imskr commented:
@praveenscience I found their images helpful & easy so taken from there.

imskr commented:
@alodahl Should i write command line installation ? Or both.

alodahl commented:
@imskr both ways would be great!  Also, when you use existing tutorials and pictures, make sure to provide a link for the source.  If you are using a lot of materials from the same article, it’s good to introduce that article before you start and describe how you’re using it.

imskr commented:
Cool!  I will do it @alodahl

imskr commented:
@alodahl I have removed the previous images and added my own images. And also i have added the installation process through command line.

alodahl commented on databases/installing-postgresql.md:
```diff
@@ -1,90 +1,131 @@
+<img src="images/step14.png" alt="step14">
 
-Build on the first information. Have apprentices guess things, do an activity, etc.
+* **Step 15** - Click on Servers > Postgre SQL 11 in the left tree.
```
Here, line 34,  74, and 131 you call it Postgre or Postgre SQL.  please change them to  PostgreSQL.

imskr replied:
done

imskr commented:
> Looks good at a glance - thanks for making this for us! I think visual examples are very helpful. However, since we are trying to avoid storing photos in the repo, could you please create a slideshow with your photos [here](https://docs.google.com/presentation/d/1m6NsBLMQCACJE2n8CmLxye44iE65RUOaTwW1EivzxmY/edit?usp=sharing) and remove them from the PR?

Yes sure! I will move those pics to the mentioned slideshow.

Kundan28 commented on databases/installing-postgresql.md:
```diff
@@ -127,5 +127,10 @@ BigSQL currently installs pgAdmin version 3 which is deprecated. It's best to ch
+## Problems
+
+**Common issues users get while installing:**
+* **Using non english characters in password** - Avoid using it.
```
Is their any restriction on using passwords other than English characters ?

imskr replied:
As found on stackoverflow. Many users find trouble while re-entering passwords other than english characters. It doesn't match sometimes. So its better to avoid it.

Kundan28 replied:
Oh, okay.

imskr commented:
@alodahl I have removed the pics from the repo and uploaded it to the google drive and added the links in the file to make it more visual. And also i have uploaded the pics to this [Slide](https://docs.google.com/presentation/d/1m6NsBLMQCACJE2n8CmLxye44iE65RUOaTwW1EivzxmY/edit?usp=sharing).

imskr commented:
@alodahl I have removed the images link from the outline and added the slide to it!
